pkg/operator/controllers/autosizednodes/autosizednodes_controller.go

```diff
@@ -0,0 +1,133 @@
+package autosizednodes
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/coreos/ignition/v2/config/util"
+	mcv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/sirupsen/logrus"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+	"github.com/Azure/ARO-RP/pkg/operator/controllers"
+)
+
+type Reconciler struct {
+	client client.Client
+
+	log *logrus.Entry
+}
+
+const (
+	configName                 = "dynamic-node"
+	AUTOSIZED_NAMESPACE string = "aro.autosizednodes"
+	AUTOSIZED_ENABLED   string = AUTOSIZED_NAMESPACE + ".enabled"
```

Comment:
#1925 got merged and it brings all these constants to consistent form. It should be something like this:

```suggestion
        # or make it exported `ControllerEnabled`, if needs to be used in another package/e2e
	controllerEnabled = "aro.autosizednodes.enabled"
```

Note that we do not use `*_NAMESPACE` constatns and string concatenation anymore.